navigation/navigation-common/src/jbMain/kotlin/androidx/navigation/NavGraph.jb.kt

```diff
@@ -47,7 +47,7 @@ public actual open class NavGraph actual constructor(navGraphNavigator: Navigato
         lastVisited: NavDestination
     ): DeepLinkMatch? {
         // First search through any deep links directly added to this NavGraph
-        val bestMatch = super.matchDeepLink(route)
```

Comment:
Why it doesn't happen in AOSP? https://github.com/androidx/androidx/blob/b4f2ba144b1330987d46dda7707499423171642b/navigation/navigation-common/src/main/java/androidx/navigation/NavGraph.kt#L117

Reply:
Because overload with `NavDeepLinkRequest` arg type (android-only)